README_human.md

```diff
@@ -11,3 +11,15 @@ This will usually mean opening a terminal and using `ssh` to connect. The genera
 ```bash
 	ssh username@my.hpc.server.edu.au
 ```
+
+## Downloading the human genome
+This step will be important when using the HISAT program to align the RNAseq reads onto the genome. This step will use the `wget` command, which allows the user to download particular files from the web. The general format of this command is:
+
+	```bash
+		wget http://website.come/files
+	```
+To download the *human genome*, enter the following command into the terminal on HPC
```

Comment:
It would be slightly more correct to say 

To download a version of the *human genome* with HiSat2 indexes, enter ...